nav2_behaviors/include/nav2_behaviors/plugins/drive_on_heading.hpp

```diff
@@ -214,6 +247,25 @@ class DriveOnHeading : public TimedBehavior<ActionT>
       node,
       "simulate_ahead_time", rclcpp::ParameterValue(2.0));
     node->get_parameter("simulate_ahead_time", simulate_ahead_time_);
+
+    nav2_util::declare_parameter_if_not_declared(
+      node, this->behavior_name_ + ".acceleration_limit",
+      rclcpp::ParameterValue(2.5));
+    nav2_util::declare_parameter_if_not_declared(
+      node, this->behavior_name_ + ".deceleration_limit",
+      rclcpp::ParameterValue(-2.5));
+    nav2_util::declare_parameter_if_not_declared(
+      node, this->behavior_name_ + ".minimum_speed",
+      rclcpp::ParameterValue(0.10));
+    node->get_parameter(this->behavior_name_ + ".acceleration_limit", acceleration_limit_);
+    node->get_parameter(this->behavior_name_ + ".deceleration_limit", deceleration_limit_);
+    node->get_parameter(this->behavior_name_ + ".minimum_speed", minimum_speed_);
+    if (acceleration_limit_ < 0.0 || deceleration_limit_ > 0.0) {
+      RCLCPP_ERROR(this->logger_,
+        "DriveOnHeading: acceleration_limit and deceleration_limit must be "
+        "positive and negative respectively");
+      throw std::runtime_error{"Invalid parameter: acceleration_limit or deceleration_limit"};
```

Comment:
If we're going to throw, we need to catch here [1] and return false to the lifecycle transition. I think though you can just fix the sign setting silently, but I'm also OK with this way as well 

[1] https://github.com/ros-navigation/navigation2/blob/6e1c85eba70ed9800b69886c15d73268d60f46e9/nav2_behaviors/src/behavior_server.cpp#L98